include/osquery/flags.h

```diff
@@ -190,11 +190,20 @@ class FlagAlias {
  * @param value The default value, use a C++ literal.
  * @param desc A string literal used for help display.
  */
+#ifdef WIN32
 #define OSQUERY_FLAG(t, n, v, d, s, e, c, h)              \
+  using ::fLS::StringFlagDestructor;                      \
```

Comment:
Does this not work on POSIX? If it does this can be reduced to remove the precompiler condition.